tests/test_merge_iterative_optimized.py

```diff
@@ -0,0 +1,732 @@
+# Authors: Valentino Constantinou <vconstan@jpl.caltech.edu>, Asitang Mishra <asitang.mishra@jpl.caltech.edu>
+# License: Apache 2.0
+
+from comparisons.iterative_optimized import Merge
```

Comment:
An import error exists here and results in tests failing, see: https://travis-ci.org/vc1492a/mieda/builds/646609867